db/seeds.rb

```diff
@@ -7,7 +7,7 @@
 )
 
 @user_one = User.create!(
-  email: 'a@b.com',
+  email: 'adopter1@example.com',
```

Comment:
Love these changes! Can you update the README, too? I believe it references emails like `a@b.com`, `b@c.com`, etc.